PSPrintSDK/OLPosterViewController.m

```diff
@@ -62,10 +61,12 @@ -(void)viewDidLoad{
     self.imageViews = [[NSMutableArray alloc] initWithCapacity:self.product.quantityToFulfillOrder];
     for (NSUInteger i = 0; i < self.product.quantityToFulfillOrder; i++){
```

Comment:
I guess this block will become redundant if we move to only a single image per Poster? Probably some tidying up in this class in general to remove redundant code we no longer need